### PR TITLE
docs: update whats-new

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: docs/whats-new.mdx [hunk 1]
```diff
@@ -19,6 +20,8 @@ Target Release Date: April, 2024
 
 **@loaders.gl/parquet**
 
+- Support compression modes: 'zstd', 'snappy', 'gzip', 'brotli', 'no_compression'.
```
Comment (ibgreen) [inline docs/whats-new.mdx:23]: Some of these were already supported? Certainly no_compression, gzip and snappy?

File: docs/whats-new.mdx [hunk 2]
```diff
@@ -19,6 +20,8 @@ Target Release Date: April, 2024
...
+- Geoparquet: support large 'Polygons' and 'MultiPolygons'. Up to 68159 vertices for 'MultiPolygon' and 20062 vertices for 'Polygon' has been tested.
```
Comment (ibgreen) [inline docs/whats-new.mdx:24]: It is a little odd to specify such numbers, it sounds like we don't understand what the limits in our implementation are.

**Review by ibgreen — APPROVED** [review]

File: docs/whats-new.mdx (hunk 2 above)
Comment (belom88) [reply docs/whats-new.mdx:24]: "Geoparquet: support large 'Polygons' and 'MultiPolygons' (> 10,000 vertices)"

Is this better?

Comment (belom88) [reply docs/whats-new.mdx:24]: Actually, threshold was 65536 vertices (16 bit)

Comment (ibgreen) [reply docs/whats-new.mdx:24]: What confused me is that we reverted the Uint16 fix.

Comment (belom88) [reply docs/whats-new.mdx:24]: You reverted but then you've fixed it back in https://github.com/visgl/loaders.gl/pull/2940 . That was lucky change in another place )

Comment (belom88) [reply docs/whats-new.mdx:24]: image

Comment (belom88) [reply docs/whats-new.mdx:24]: So the reverted PR is not neede anymore

File: docs/whats-new.mdx (hunk 1 above)
Comment (belom88) [reply docs/whats-new.mdx:24]: I don't remember right now. Daria made fixes for few types of compressions.

**Pull request merged**